packages/builder/.env.example

```diff
@@ -1,6 +1,9 @@
 # development for local, production for prod builds
 REACT_APP_ENV=development
 
+# removing this will enable the default production or staging chains
+REACT_APP_CHAINS=dev1,dev2
```

Comment:
> This seems easy to misunderstand down the road, especially when seeing it in code without the benefit of the comment. It sounds like the list of enabled chains, so an empty variable would mean no chains, like in the indexer.

@bard @vacekj  my idea is to set the chains there explicitly but using this empty string trick in the transition, so that we can merge the PR (which is deployed automatically to prod), update all the env vars in prod and test that everything works correctly, and then remove the allowed empty string so that we always need to specify them.
but this is just the first step of the local dev env, and by the end of the work I want to have the same configuration for all the 3 apps

Reply:
updated it as you proposed @bard but still with ha temporary trick to allow optional vars until we globally use getConfig and we remove the verify-env task in favour of something similar calling directly getEnv

Reply:
a compile-time check will still be needed to ensure env vars are set during build and development. Paired with getConfig that returns non-optional strings (in a future PR), this ensures that the environment is always correct (or at least the env vars are set - doesn't verify the correctness of api keys ofc etc.), and greatly reduces having to deal with optional values in the codebase.